src/sass/core/transform.scss

```diff
@@ -63,7 +55,17 @@ $-origin-opts: (
 			}
 		}
 
-		@each $rv in $-scale-range {
+		@each $rv in get.decimal-range(0, 1) {
+			// $rv : 0, 0.1, 0.2, 0.25, 0.3, 0.4, 0.5, 0.6, 0.7, 0.75, 0.8, 0.9, 1,
+			.scale-#{get.abs($rv * 100)} {
+				@include mixins.variants($composed...) {
+					--transform-scale: #{$rv};
+				}
+			}
+		}
+
+		@each $rv in get.decimal-range(1, 1) {
+			// $rv : 1.1, 1.2, 1.25, 1.3, 1.4, 1.5, 1.6, 1.7, 1.75, 1.8, 1.9, 2,
 			.scale-#{get.abs($rv * 100)} {
 				@include mixins.variants($composed...) {
 					--transform-scale: #{$rv};
```

Comment:
`$-scale-range` values which is deleted, increase by 0.1 from 0 to 2. 
The `decimal-range()` function, however, can do only from 0 to 1.
So I created two different `@each` The first one starts from 0, the second starts from 1.

And I want to combine these two `@each` as they have same code inside.
Can I do like this?
```
@for $i from 0 through 1 {
@each $rv in get.decimal-range(0, $i) {
			.scale-#{get.abs($rv * 100)} {
				@include mixins.variants($composed...) {
					--transform-scale: #{$rv};
				}
			}
}
```

Or is there better way to do it making the values from 0 to 2?

Reply:
Well the basic idea is that we can do something like `utils.list-of(decimal-range(...), decimal-range(...))`. That way decimal ranges only worry about incrementing from like 0 to 1, or 1 to 2, but doesn’t go from 0 to 2. That’s probably how I would get around this.